From: Caleb Connolly <caleb.connolly@linaro.org>
Date: Sun, 19 May 2024 22:48:33 +0200
Subject: [PATCH] sdm845.config: add missing ASCII encoding option

needed for fat32 boot partitions.

Signed-off-by: Caleb Connolly <caleb.connolly@linaro.org>
(cherry picked from commit 3e87b306feff5d887ad37dc067ebc9929077dcb2)
Link: https://gitlab.com/sdm845-mainline/linux
Signed-off-by: Richard Acayan <mailingradian@gmail.com>
---
 arch/arm64/configs/sdm845.config | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/sdm845.config b/arch/arm64/configs/sdm845.config
index 2b5e265..e102374 100644
--- a/arch/arm64/configs/sdm845.config
+++ b/arch/arm64/configs/sdm845.config
@@ -265,6 +265,7 @@ CONFIG_TMPFS_XATTR=y
 CONFIG_BLK_DEV_DM=y
 CONFIG_DM_CRYPT=y
 CONFIG_BINFMT_MISC=m
+CONFIG_NLS_ASCII=y
 # CONFIG_USB_MASS_STORAGE is not set
 
 CONFIG_NF_TABLES=m
@@ -997,4 +998,4 @@ CONFIG_SND_SOC_WCD938X_SDW=n
 CONFIG_MMC_SDHCI_OF_DWCMSHC=n
 CONFIG_IOMMU_IO_PGTABLE_DART=n
 CONFIG_MEMORY_HOTPLUG=n
-CONFIG_MELLANOX_PLATFORM=n
\ No newline at end of file
+CONFIG_MELLANOX_PLATFORM=n
